lua/starfall/sflib.lua

```diff
@@ -562,7 +536,7 @@ if SERVER then
         if ply and not IsValid( ply ) then return end
 
         net.Start( "starfall_addnotify" )
-	net.WriteString( msg )
+		net.WriteString( msg )
```

Comment:
This, and L556 seem opposites. Either have indenting on for writes after a start, or off. I'm leaning towards off as other writes in context are unindented.